lib/models/disk.rb

```diff
@@ -48,7 +49,7 @@ def api_format
     {
       "id": remote_id,
       "name": name,
-      "maximum_size_bytes": size ? size : 0, # Default to 0 if nil, otherwise API throws error
+      "storage_bytes": size ? size : 0, # Default to 0 if nil, otherwise API throws error
```

Comment:
We updated some of the places (disk size and memory I think) where there were "maximums" because they don't really make sense when talking about configuration. As an example, when you go to buy a hard drive, you don't ask for one with a 200 GB "maximum size" - you just ask for a 200 GB hard drive ;)
Also, it sorta implies there are "minimum" sizes as well, which is not the case for our API.